From 1697553b33407927669f173dfd8c815cf0f113e1 Mon Sep 17 00:00:00 2001
From: dirk <dirk@git.imagemagick.org>
Date: Mon, 11 May 2015 11:02:38 +0000
Subject: [PATCH] Jpeg images no longer have pixels per inch as a default value
 for density units when the density is not set.

Debian-bug: https://bugs.debian.org/763799
Applied-upstream: 6.9.1.3
---
 coders/jpeg.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/coders/jpeg.c b/coders/jpeg.c
index 2263eaf..d1310f6 100644
--- a/coders/jpeg.c
+++ b/coders/jpeg.c
@@ -2155,7 +2155,6 @@ static MagickBooleanType WriteJPEGImage(const ImageInfo *image_info,
     jpeg_info.data_precision=8;
   else
     jpeg_info.data_precision=BITS_IN_JSAMPLE;
-  jpeg_info.density_unit=1;
   if (image->debug != MagickFalse)
     (void) LogMagickEvent(CoderEvent,GetMagickModule(),
       "Image resolution: %.20g,%.20g",floor(image->x_resolution+0.5),
@@ -2171,7 +2170,6 @@ static MagickBooleanType WriteJPEGImage(const ImageInfo *image_info,
       /*
         Set image resolution units.
       */
-      jpeg_info.density_unit=(UINT8) 0;
       if (image->units == PixelsPerInchResolution)
         jpeg_info.density_unit=(UINT8) 1;
       if (image->units == PixelsPerCentimeterResolution)
